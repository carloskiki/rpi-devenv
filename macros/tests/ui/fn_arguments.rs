#[macros::main]
pub fn kernel_main(a: u32) {}

pub fn main() {}
