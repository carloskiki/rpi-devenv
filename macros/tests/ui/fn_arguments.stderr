error: main function cannot have parameters
 --> tests/ui/fn_arguments.rs:2:20
  |
2 | pub fn kernel_main(a: u32) {}
  |                    ^^^^^^
